content/tracing/languages/java.md

```diff
@@ -0,0 +1,271 @@
+- link: "/tracing/"
+  tag: "Documentation"
+  text: APM (Tracing)
+---
+
+## Compatibility
+
+We officially support Java 1.7 and higher of both Oracle JDK and OpenJDK.  To see what web frameworks, libraries, and datastores we support, see the Integrations section below.
+
+## Installation and Getting Started
+
+To begin tracing applications written in any language, first [install and configure the Datadog Agent](https://docs.datadoghq.com/tracing#installing-the-agent).
+
+Next, download `dd-java-agent.jar` that contains the agent class files:
+
+```shell
+wget -O dd-java-agent.jar 'https://search.maven.org/remote_content?g=com.datadoghq&a=dd-java-agent&v=LATEST'
+```
+
+Finally, add the following JVM argument when starting your application in your IDE, your Maven or Gradle application script, or your `java -jar` command:
+
+```
+-javaagent:/path/to/the/dd-java-agent.jar
+```
+
+## Configuration
+
+The tracer is configured using System Properties and Environment Variables as follows:
+
+| Config             | System Property       | Environment Variable      |  Default           | Description |
+|:------------------ |:--------------------- |:------------------------- |:------------------ |:----- |
+| service.name       | dd.service.name       | DD_SERVICE_NAME           | `unnamed-java-app` | The name of a set of processes that do the same job. Used for grouping stats for your application.|
+| writer.type        | dd.writer.type        | DD_WRITER_TYPE            | `DDAgentWriter`    | Default value sends traces to the trace agent. Configuring with `LoggingWriter` instead writes traces out to the console. |
+| agent.host         | dd.agent.host         | DD_AGENT_HOST             | `localhost`        | Hostname for where to send traces to. If using a containerized environment, configure this to be the host ip.  See our [docker docs](https://docs.datadoghq.com/tracing/docker/) for additional detail. |
+| agent.port         | dd.agent.port         | DD_AGENT_PORT             | `8126`             | Port number the agent is listening on for configured host. |
+| priority.sampling  | dd.priority.sampling  | DD_PRIORITY_SAMPLING      | `false`            | Changes how the client and agent sample traces. See [Sampling / distributed tracing](#sampling-distributed-tracing) section for details. |
+
+**Note**: If the same key type is set for both, the system property configuration takes priority.
+
+## Manual Instrumentation 
+
+Before instrumenting your application, review Datadog’s [APM Terminology](/tracing/terminology/) and familiarize yourself with the core concepts of Datadog APM.
+
+If you aren't using a [supported framework instrumentation](#integrations), or you would like additional depth in your application’s traces, you may want to to manually instrument your code.  
+Do this either using the [Trace annotation](#trace-annotation) for simple method call tracing or with the [OpenTracing API](#opentracing-api) for complex tracing.
```

Comment:
I think this can be on the same line as the previous sentence.

Reply:
👍